feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
It should be ideally possible. But looking at the data pipeline - it'll need some more investigation.
`stats_services.get_top_state_rule_answers_multi` calls `stats_domain.StateRuleAnswerLog.get_multi_by_multi_explorations` internally which in turn calls `stats_models.StateRuleAnswerLogModel.get_or_create_multi_for_multi_explorations` where the actual fetch from the datastore seems to happen. Looking at the code for `stats_models.StateRuleAnswerLogModel.get_or_create_multi_for_multi_explorations`: https://github.com/oppia/oppia/blob/2aeb715c5595c1676c8a56fe67449959c01dca7d/core/storage/statistics/gae_models.py#L94-L145 .. it isn't immediately clear to me how to limit to the top N results by frequency!